extensions/validation/src/test/java/org/mule/extension/validation/BasicValidationTestCase.java

```diff
@@ -149,22 +156,17 @@ public void isFalse() throws Exception
     public void notEmpty() throws Exception
     {
         final String flowName = "notEmpty";
-
```

Comment:
I understand that for splitting setup, execution and assertions but not for declaring a variable. Anyway, we should agree to split these sections always or never do it.
For this particular case I'm creating a constant, which seems to be the correct solution. Is that OK for you?